.travis.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   include:
     - stage: Run tests
       script:
-        - find . -name vendor -prune -o -name Makefile -execdir make test \;
+        - find examples -name Makefile -print | xargs -I% sh -c 'make -C $(dirname %) test'
```

Comment:
It seems more future-proof to search '.' for Makefiles.

Side note: when did this repo grow Makefiles??

Reply:
I changed it to search `examples` rather than `.` since `examples` is the only dir with any Makefiles currently. It feels odd searching for any Makefile only to have to prune out the directories we don't want (in this case the `vendor` dir).

I imagine if somebody wants to add an extra Makefile in the future to some other directory,  they'll modify this script to include the new dir. This way testing on the CI is opt-in, rather than opt-out. Does explaining the intention change your mind, or is that opposite of what you meant by future-proofing? 

And I can't answer your second question! 😅 I only caught this because I wanted to submit a feature PR but wasn't comfortable with the failing tests so I wanted to fix them first.

Reply:
I tend to agree with @lavalamp, i.e. pruning is better as it avoid that somebody adds a non-example Makefile which is silently ignored.

Reply:
Okiedoke - I've updated it!